src/Widgets/TerminalWidget.vala

```diff
@@ -171,6 +173,42 @@ namespace Terminal {
                 return false;
             });
 
+            scroll_event.connect ((event) => {
+                if ((event.state & Gdk.ModifierType.CONTROL_MASK) > 0) {
+                    switch (event.direction) {
+                        case Gdk.ScrollDirection.UP:
+                            increment_size ();
+                            return Gdk.EVENT_STOP;
+
+                        case Gdk.ScrollDirection.DOWN:
+                            decrement_size ();
+                            return Gdk.EVENT_STOP;
+
+                        case Gdk.ScrollDirection.SMOOTH:
+                            double delta_x, delta_y;
+                            event.get_scroll_deltas (out delta_x, out delta_y);
```

Comment:
If you're only interesting in `delta_y` I think you can just do 
```suggestion
                            var delta_y = event.delta_y;
```

Reply:
Thanks - done